docs/conf.py

```diff
@@ -14,7 +14,7 @@
 
 # -- Project information -----------------------------------------------------
 
-project = "arrow"
+project = "Arrow 🏹"
```

Comment:
Is it safe to have an emoji in the conf file?

Reply:
It breaks as it is now when the docs are built with Python 2.7, but I fixed it by changing it to `u"Arrow 🏹"` :). Docs build and display very nicely!